src/warp/WarpRouteDeployer.ts

```diff
@@ -70,13 +73,15 @@ export class WarpRouteDeployer {
   }
 
   async deploy(): Promise<void> {
-    const { configMap, baseToken } = await this.buildHypERC20Config();
+    const { configMap, baseToken } = await this.buildHypTokenConfig();
```

Comment:
This might be out of scope here, but I suspect @yorhodes has opinions on whether we should have a WarpRouteDeployer when we have the underlying deployers

Reply:
This handles different things, like reading/writing token configs. There may be some overlap and I'm open to DRY-ing up the two. A step towards that would be to move this stuff to the monorepo and couple it more tightly with the other layers.